### PR TITLE
Export visualisation feature 

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Review by admin-vizcentric — APPROVED** [review]

File: src/main/webapp/app/entities/flair-bi/flair-bi.controller.js [hunk 1]
```diff
@@ -1126,6 +1126,7 @@
                 );
                 var filterUrl = filterParametersService.getFilterURL(vm.dimensions);
                 url = url + "&isExport=true" + filterUrl;
+                $window.localStorage.setItem(visualMetadata.id,JSON.stringify( visualMetadata.fields));
```
Comment (khan-j) [inline src/main/webapp/app/entities/flair-bi/flair-bi.controller.js:1129]: make sure you clean the local storage when logged out

File: src/main/webapp/app/entities/flair-bi/fullscreen/flair-bi-fullscreen.html [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
-            </div>
+            </button>
+            
         </div>
-
     </div>
-
     <div class="row">
-        <div style="min-height:92.5vh;top: 0px;" class="col-md-12 widget-container-share-link" data="vm.visualMetadata"
+        <div style="min-height:92.5vh;top: 0px;padding-left: 0px;" class="col-md-12 widget-container-share-link" data="vm.visualMetadata"
```
Comment (khan-j) [inline src/main/webapp/app/entities/flair-bi/fullscreen/flair-bi-fullscreen.html:25]: place this css in scss file

File: src/main/webapp/app/entities/flair-bi/fullscreen/flair-bi-fullscreen.html [hunk 3]
```diff
@@ -1,40 +1,39 @@
...
             ng-style="{'background-color': vm.visualMetadata.bodyProperties.backgroundColor}" visualization-render-grpc
             can-build="vm.visualMetadata.canBuild()"
             id="content-{{vm.visualMetadata.id || vm.visualMetadata.visualBuildId}}"
             widget="{{vm.visualMetadata.metadataVisual.functionname}}" reload="vm.change" datasource="vm.datasource">
         </div>
 
         <div class="col-md-2">
-            <div id="slider" class="flair-scroll-gray" ng-style="vm.thinbarStyle">
+            <div id="slider" style="margin: auto;" class="flair-scroll-gray" ng-style="vm.thinbarStyle">
```
Comment (khan-j) [inline src/main/webapp/app/entities/flair-bi/fullscreen/flair-bi-fullscreen.html:33]: place css in scss file

**Review by khan-j — COMMENTED** [review]
pls make the suggested changes

**Pull request merged**